### PR TITLE
Add a setup.py for a source distribution

### LINKED ISSUE #27: Prevent installing on systems that are not x86_64
We don't have any checks for x86_64. We should have our wheels restrict to only the platforms that we support. Ideally with names like this: numpy-1.13.3-cp35-cp35m-manylinux1_x86_64.whl

### PR DESCRIPTION
**Pull request by znation**
We will use this one for erroring out on unsupported platforms, by
uploading it as an egg (source distribution). If a user's pip install
falls back to the egg (no compatible wheel) they will see this error
message.

* Fixes #27

### CODE REVIEW

**Comment by gustavla** [comment]
Is the plan to add a wheel with this for the current 4.2 and not for any future releases? That makes sense to me, just double-checking that plans.

**Comment by znation** [comment]
@gustavla Yes, that was my plan.

File: src/unsupported_python/setup.py [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+        * Outdated pip version (try `pip install -U pip`).
+
+        ==================================================================================
+
+        """)
+        sys.stderr.write(msg)
+        sys.exit(1)
+
+if __name__ == '__main__':
+    from distutils.util import get_platform
```
Comment (gustavla) [inline src/unsupported_python/setup.py]: Why is this imported?

Comment (znation) [reply src/unsupported_python/setup.py]: It doesn't need to be. Good catch.

**Pull request merged**